pkg/controller/common/helpers.go

```diff
@@ -1,14 +1,22 @@
 package common
 
 import (
-	ignv2_2types "github.com/coreos/ignition/config/v2_2/types"
+	igntypes "github.com/coreos/ignition/config/v3_0/types"
 )
 
-// NewIgnConfig returns an empty ignition config with version set as 2.2.0
-func NewIgnConfig() ignv2_2types.Config {
-	return ignv2_2types.Config{
-		Ignition: ignv2_2types.Ignition{
-			Version: "2.2.0",
+// NewIgnConfig returns an empty ignition config with version set as 3.0.0
+func NewIgnConfig() igntypes.Config {
+	return igntypes.Config{
+		Ignition: igntypes.Ignition{
+			Version: "3.0.0",
```

Comment:
this can be moved to `igntypes.MaxVersion.String()` w/o hardcoding the string itself

Reply:
done.